README.md

```diff
@@ -1,3 +1,73 @@
 # Remote environment client
 
-This 
+A command line tool to help with using Continuous Pipe as a remote development environment.
+
+This helps to set up Kubectl, create, build and destroy remote environments and keep files
+in sync with the local filesystem.
+
+## Installation
+
+
+## Setup
+
+To start using this tool for a project, run the `setup` command from the project root.
+ This will install kubectl if you do not have it installed already. It will then 
+ ask a series of questions to get the details for the project set up. 
+ 
+## Create
+
+The `create` command will push changes the branch you have checked out locally to your remote 
+ environment branch. Continuous Pipe will then build the environment. YOu can use the [Continuous Pipe admin
+ site](http://ui.continuouspipe.io/) to see when the environment has finished building and 
+ to find its IP address.
+ 
+## Watch
+ 
+ The `watch` command will sync changes you make locally to a container that's part of the remote environment.
```

Comment:
Maybe that before:

**Note:** you need to wait the remote environment to be created before running this command.